Modules/Core/Common/test/itkShapedImageNeighborhoodRangeGTest.cxx

```diff
@@ -350,7 +350,7 @@ TEST(ShapedImageNeighborhoodRange, IteratorsCanBePassedToStdVectorConstructor)
 // second argument of std::reverse (which requires bidirectional iterators).
 TEST(ShapedImageNeighborhoodRange, IteratorsCanBePassedToStdReverseCopy)
 {
-  using PixelType = unsigned char;
+  using PixelType = unsigned short;
```

Comment:
FYI:  Initially I instrumented this with gcc pragma commands to ignore the warning, but the problem seems to be a false positive warning due to assuming that the range is a string.   Instead of suppressing the false positive due to processsing unsigned char, just change the type to unsigned short for this simple test case.


I looked carefully at the code in a debugger, and it appears that the behavior is correct for these containers.